net-im/spectrum2/spectrum2-2.0.6.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils user
+
+DESCRIPTION="An open source instant messaging transport"
+HOMEPAGE="https://www.spectrum.im"
+SRC_URI="https://github.com/SpectrumIM/spectrum2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc frotz irc mysql postgres purple sms sqlite test twitter xmpp"
+REQUIRED_USE="|| ( mysql postgres sqlite )"
+
+DEPEND="dev-libs/boost
+	dev-libs/expat
+	dev-libs/libxml2
```

Comment:
Why does it use two different XML libraries?

Reply:
I am not sure, I see both are used, if found by CMakeLists.txt. `ldd` confirms linked against both.
But it also works, if linked against only one of them. I made a new use flag `expat`.

Reply:
Please investigate the code. It may indeed need only one of them but accidentally link to both. But it may also have independent features that use either (e.g. some old code using expat and new code using libxml2, or the other way around).